File: Extras x RetosJS/4 | Numeros Mayores/4-Extra.js
/* 
    * Contar Números Mayores que un Valor
    * Cuenta cuántos números en un arreglo son mayores que un valor dado.
*/

// ? Solución

function contarMayoresQue(numeros, x) {
	let cantidadMayor = 0;
	for (let i = 0; i <= numeros.length; i++) {
		if (numeros[i] > x) {
			cantidadMayor = cantidadMayor + 1;
		}
	}
	return cantidadMayor;
}

// Todo: Pruebas

console.log(contarMayoresQue([1,4,7,5,9,7,6], 5)) // Resultado: 4

File: Extras x RetosJS/4 | Numeros Mayores/4-Extra.py
# Contar Números Mayores que un Valor
# Cuenta cuántos números en un arreglo son mayores que un valor dado.

number = [1,2,3,4,5,6,7,8,9,10]
numberComparative = 5
counter = 0

for i in range(len(number)):
    if number[i] > numberComparative:
        counter = counter + 1

print(counter)